x-pack/plugins/security_solution_serverless/server/endpoint/services/metering_service.ts

```diff
@@ -124,6 +115,21 @@ export class EndpointMeteringService {
       },
     };
   }
+
+  private setTier(config: ServerlessSecurityConfig) {
+    if (this.tier) {
```

Comment:
is it safe to assume that kibana will be restarted if tier changes?

Reply:
Right now, I think that is true, however, that could change in the future.  I thought that there was a product tier service that used observables internally and ensured that it remained in sync - but maybe I'm wrong 🤔 

Maybe reach out to @semd ??